hpu-utils/profile_forward.py

```diff
@@ -0,0 +1,113 @@
+###############################################################################
+# Copyright (C) 2024 Habana Labs, Ltd. an Intel Company
+###############################################################################
+
+import argparse
+import torch
+import os
```

Comment:
As a convenience you can add: `os.environ['VLLM_SKIP_WARMUP']='true'` somewhere at the top before loading vllm.
This way the users don't need to remeber to do that manually.